### DISCUSSION TITLE
What happened to the Chinese company that were planning to mass produce build kits?

### DISCUSSION THREAD

**Discussion by abceleung** [hrvach/deskhop#53] [Q&A]
I am pretty sure that README mentioned a Chinese company that was interested to mass produce build kits for this project. Now the README doesn't include this part.
What happened? Did they give up?

**Comment by abceleung** [comment]
Oops, just saw it in the README. Somehow I missed it.

**Comment by hrvach** [comment]
I'm guessing they have a backlog due to the recent New Year holiday, waiting on a small case lid mod to be verified as their prints were a slightly looser fit than what I was seeing on my printer. I'll definitely update the README with the links as soon as I hear back from them.
